packages/cli/src/utils/generate.js

```diff
@@ -169,7 +169,7 @@ export async function snapshot ({ globbyOptions, ignore, rootDir }) {
   const files = await globby('**/*.*', {
     ...globbyOptions,
     ignore,
-    cwd: rootDir,
+    cwd: path.normalize(rootDir),
```

Comment:
I think it would be better normalizing the path directly in `@nuxt/config` ([here](https://github.com/nuxt/nuxt.js/blob/3c10f176e69b29a88daaae6fb353b9c5fc198031/packages/config/src/options.js#L70)) otherwise we will have inconsistent behavior.

Reply:
I would also check whether it works with `upath.normalize` as that is what we use elsewhere.

Reply:
One downside of upath, is that it seems not supporting windows network mounts (that might be currently used by nuxt2 users). If `path.normalize` solving issue with `globby` is less risky change (till we have `unpath` for better normalization)